.github/workflows/ai-review.yaml

```diff
@@ -2,19 +2,44 @@ name: AI review
 
 on:
   pull_request:
-    branches:
-    - master
 
+env: 
+  GITHUB_REFSPEC: pull/${{ github.event.number }}/head:FETCH_HEAD
+  REVIEW_SERVER: http://10.255.248.1
+  
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - name: Extract all keywords from package.json
-        uses: sergeysova/jq-action@v2
-        id: keywords
-        with:
-          cmd: '{"a":1, "b": "2"}'
-          multiline: true
+      - name: Run review
+        run: |
+          curl --silent \
+            -H "Authorization: ${{ secrets.REVIEW_TOKEN }}" \
+            -H "Content-Type: application/json" \
+            ${REVIEW_SERVER}/review  -d '{"refspec": "$GITHUB_REFSPEC", "project": "$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"}'
```

Comment:
**Suggestion:** The curl command in the 'Run review' step is missing a POST method (-X POST). Without specifying the method, curl will use GET by default, which may not be what you want when sending data to the review server.
```suggestion
          curl --silent \
          	-X POST \
          	-H "Authorization: ${{ secrets.REVIEW_TOKEN }}" \
          	-H "Content-Type: application/json" \
          	${REVIEW_SERVER}/review  -d '{"refspec": "$GITHUB_REFSPEC", "project": "$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"}'
```